src/shared/constants.ts

```diff
@@ -93,13 +93,13 @@ const networkMainnet: NetworkConfiguration = {
   chain: {
     stacks: {
       blockchain: 'stacks',
-      chainId: ChainID.Mainnet,
-      url: HIRO_API_BASE_URL_MAINNET,
+      stacksChainId: ChainID.Mainnet,
+      stacksUrl: HIRO_API_BASE_URL_MAINNET,
```

Comment:
Is it necessary to change these names?

Reply:
It was cleaner for me to distinguish it from the bitcoin properties, but I can revert this if you think it is unnecessary.